AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -857,7 +857,7 @@ - (void)layoutSubviews
   _nextLayoutInvalidationStyle = ASCollectionViewInvalidationStyleNone;
   switch (invalidationStyle) {
     case ASCollectionViewInvalidationStyleWithAnimation:
-      if (!_superPerformingBatchUpdates) {
+      if (0 == _superBatchUpdateCount) {
```

Comment:
@appleguy @Adlai-Holler What is the opinion about the code style? I personally would prefer `0 == _superBatchUpdateCount`, but we should find a way that works for all and the future